README.md

```diff
@@ -78,8 +77,29 @@ And the installed version of python will be printed.
 
 ---
 
+<details><summary>Step 3 : Installing Git </summary>
 
-<details><summary>Step 3 : Creating Project Directory </summary>
+Click on [Download Git](https://git-scm.com/downloads)
+
+</details>
+
+
+---
+
+
+<details><summary>Step 4 : Fork the Repository </summary>
+
+Click on ![Fork](https://user-images.githubusercontent.com/63921263/110382285-b07bba80-8080-11eb-8407-d354849c1753.png)
+to fork [this Repo](https://github.com/chiraag-kakar/sharenlearn)
+
+</details>
+
+
```

Comment:
Added Link to download the Git  to maintain inheritance as readme contains links to download the other prerequisites.